garden/src/components/ScrollToTop.tsx

```diff
@@ -5,7 +5,7 @@ export default function ScrollToTop() {
   const { pathname } = useLocation();
 
   useEffect(() => {
-    window.scrollTo(0, 0);
+    // window.scrollTo(0, 0);
```

Comment:
Did you mean to comment this out? Seems like it would make the scroll to the top functionality not work?

Reply:
You're right! This line was giving me a warning by our test framework, I meant to uncomment it. For now we'll live with the warning